docs/APIReference.md

```diff
@@ -1,3 +1,116 @@
-# API Reference
+<!-- Generated with Stardoc: http://skydoc.bazel.build -->
+
+<a name="#dotnet_wrapper"></a>
 
-TODO: generate with stardoc (issue [#17](https://github.com/Brightspace/rules_csharp/issues/17)).
+## dotnet_wrapper
```

Comment:
Could we hide this from the output?
I get that `dotnet_wrapper` needs to be public for Reasons(TM) but I don't think it's an API we want to document.

Reply:
The rule `stardoc` offers a `symbol_names` attribute where you can list all of the symbols you want to have documented.

```bazel
stardoc(
    name = "my-docs",
    input = "doc_hub.bzl",
    out = "docs.md",
    symbol_names = ["foo_rule", "bar_rule", "baz_rule"],
)
```

I think we would want to restrict the list using that.